[2025-01-25T19:16:00.016+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-25T19:16:00.033+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: prim_pipelines.load_from_file manual__2025-01-25T18:48:55.829990+00:00 [queued]>
[2025-01-25T19:16:00.041+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: prim_pipelines.load_from_file manual__2025-01-25T18:48:55.829990+00:00 [queued]>
[2025-01-25T19:16:00.042+0000] {taskinstance.py:2866} INFO - Starting attempt 3 of 3
[2025-01-25T19:16:00.055+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): load_from_file> on 2025-01-25 18:48:55.829990+00:00
[2025-01-25T19:16:00.062+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=642) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-25T19:16:00.063+0000] {standard_task_runner.py:72} INFO - Started process 657 to run task
[2025-01-25T19:16:00.063+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'prim_pipelines', 'load_from_file', 'manual__2025-01-25T18:48:55.829990+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp49srdy4i']
[2025-01-25T19:16:00.064+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask load_from_file
[2025-01-25T19:16:00.108+0000] {task_command.py:467} INFO - Running <TaskInstance: prim_pipelines.load_from_file manual__2025-01-25T18:48:55.829990+00:00 [running]> on host c96922973881
[2025-01-25T19:16:00.186+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='prim_pipelines' AIRFLOW_CTX_TASK_ID='load_from_file' AIRFLOW_CTX_EXECUTION_DATE='2025-01-25T18:48:55.829990+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-25T18:48:55.829990+00:00'
[2025-01-25T19:16:00.187+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-25T19:16:00.289+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-25T19:16:00.302+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-25T19:16:00.303+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=prim_pipelines, task_id=load_from_file, run_id=manual__2025-01-25T18:48:55.829990+00:00, execution_date=20250125T184855, start_date=20250125T191600, end_date=20250125T191600
[2025-01-25T19:16:00.358+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-25T19:16:00.381+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-25T19:16:00.383+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
